.github/workflows/publish-release.yml

```diff
@@ -227,3 +227,42 @@ jobs:
         sdk: Flutter/Dart
         webhook-url: ${{ secrets.SLACK_RELEASES_WEBHOOK }}
         version: ${{ needs.prepare-packages.outputs.version }}
+
+    - name: Update Changelog
+      run: |
+        echo "## vNext (TBD)
+
+        **This project is in the Beta stage. The API should be quite stable, but occasional breaking changes may be made.**
+
+        ### Enhancements
+        * None
+
+        ### Fixed
+        * None
+
+        ### Compatibility
+        * Realm Studio: 12.0.0 or later.
+
+        ### Internal
+        * Using Core x.y.z.
+        " | cat - CHANGELOG.md >> temp
+        mv temp CHANGELOG.md
+      shell: bash
+
+    - name: Create vNext PR
+      id: vnext-pr
+      uses: peter-evans/create-pull-request@7380612b49221684fefa025244f2ef4008ae50ad
+      with:
+        branch: prepare-vnext
+        title: Prepare for vNext
```

Comment:
Should we name it `Add vNext header` or something. Since we are not preparing for a release but just adding a header on top of the changelog